kernel/h1/src/hil/flash/smart_program.rs

```diff
@@ -110,7 +110,7 @@ pub fn div_round_up(numerator: u64, denominator: u64) -> u64 {
 }
 
 fn set_program_timeout<'a, A: Alarm<'a>>(alarm: &A, timeout_nanoseconds: u32) {
-    alarm.set_alarm(alarm.now().wrapping_add(
-        div_round_up(A::Frequency::frequency() as u64 * timeout_nanoseconds as u64,
-                     1_000_000_000) as u32));
+    alarm.set_alarm(alarm.now(),
+        (div_round_up(A::Frequency::frequency() as u64 * timeout_nanoseconds as u64,
+                     1_000_000_000) as u32).into());
```

Comment:
Nit: Weird indentation.  Maybe run `rustfmt` after, as a separate PR?

Reply:
Yes, I'll run rustfmt in a separate commit. Thanks for reminding me.